frontend/src/components/AccountReceiveTableWithdrawConfirmation.vue

```diff
@@ -182,18 +182,16 @@ export default defineComponent({
 
     const useCustomFee = ref<boolean>(false);
     const toggleCustomFee = () => (useCustomFee.value = !useCustomFee.value);
-    const formattedCustomFee = ref<BigNumber | string>('0'); // gas price in Gwei
+    const formattedCustomTxCostGwei = ref<BigNumber | string>('0'); // gas price in Gwei
     // the custom gas price; determines what gas price is actually used when withdrawing
-    const customGasInWei = computed(() => {
-      const customGasInGwei = formattedCustomFee.value ? formattedCustomFee.value : 0;
+    const customGasPriceInWei = computed(() => {
+      const customGasInGwei = formattedCustomTxCostGwei.value ? formattedCustomTxCostGwei.value : 0;
       return BigNumber.from(customGasInGwei).mul(10 ** 9);
     });
-    const customFeeInWei = computed(() => {
-      return gasLimit.value.mul(customGasInWei.value);
+    const customTxFeeInWei = computed(() => {
+      return gasLimit.value.mul(customGasPriceInWei.value);
     });
```

Comment:
Resisting the urge to complain about this one liner taking up 3 lines 😝 (because it was already like that prior to this PR)